docs/weave-tutorial/02-domain.md

```diff
@@ -0,0 +1,30 @@
+---
+id: domain 
+title: Defining the Domain 
+sidebar_label: Domain 
+---
+
+The first thing we consider is the data we want to store (the state). After that, we can focus on the messages, which trigger state transitions. All blockchain state must be stored in our merkle-ized store, that provide validity hashes and proofs. This is exposed to the application as a basic key-value store, which also allows in-order iteration over the keys. On top of this, we have built some tools like secondary indexes and sequences, similar to how [storm adds an ORM](https://github.com/asdine/storm#simple-crud-system) on top of [boltdb’s key-value store](https://github.com/boltdb/bolt#using-buckets). We have avoided struct tags and tried to type as strictly as we can (without using generics).
+
+We are building an Orderbook application. This topic is quite hot at the time of writing this tutorial.
```

Comment:
`Orderbook or Order book? In Line 10 it's Orderbook, in Line 11 it's two words.`